src/skin/legacyskinparser.cpp

```diff
@@ -660,7 +660,14 @@ QWidget* LegacySkinParser::parseWidgetStack(QDomElement node) {
 }
 
 QWidget* LegacySkinParser::parseSizeAwareStack(QDomElement node) {
-    WSizeAwareStack* pStack = new WSizeAwareStack(m_pParent);
+    ControlObject* pCurrentPageControl = NULL;
+    QString currentpage_co = node.attribute("currentpage");
+    if (currentpage_co.length() > 0) {
+        ConfigKey configKey = ConfigKey::parseCommaSeparated(currentpage_co);
+        pCurrentPageControl = controlFromConfigKey(configKey, false, NULL);
+    }
+
+    WSizeAwareStack* pStack = new WSizeAwareStack(m_pParent, pCurrentPageControl);
```

Comment:
Copy/paste the code from WWidgetStack that checks the created flag and if the CO was created then parent the CO to the stack.